Entropy1729/aleo_wallet
# Back - Add Dockerfile for the API

### Overview
Dockerfile for the Elixir REST API. Based on [Deploying with Releases](https://hexdocs.pm/phoenix/releases.html#containers) for phoenix applications.

### Build and Run
To test this dockerfile you can run:
```
cd api
docker build -t aleo_wallet_api .
docker run -p 5000:4000 aleo_wallet_api
```
The server should be running now, try an endpoint if you want:
```
curl -X POST http://127.0.0.1:5000/account
```

### Notes

This Dockerfile clones and builds the [Aleo SDK](https://github.com/AleoHQ/aleo). This should be avoided in the future when the Aleo CLI is updated in [crates.io](https://crates.io/)
